src/main/java/hudson/plugins/sshslaves/SSHLauncher.java

```diff
@@ -881,8 +881,11 @@ public Boolean call() throws InterruptedException {
             Boolean res;
             try {
                 res = results.get(0).get();
-            } catch (Exception e) {
+            } catch (Throwable e) {
                 res = Boolean.FALSE;
+                System.out.println(e.getMessage());
+                listener.getLogger().println("ECHO");
```

Comment:
Looks like forgotten debugging statements...

Reply:
yep, it is removed on latest commit ;)